packages/commerce-sdk-react/package.json

```diff
@@ -61,13 +61,13 @@
     "@types/react-dom": "^17.0.19",
     "@types/react-helmet": "^6.1.6",
     "cross-env": "^5.2.1",
-    "internal-lib-build": "^2.7.3",
+    "internal-lib-build": "^2.7.4-preview.0",
     "jsonwebtoken": "^8.5.1",
     "nock": "^13.3.0",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-helmet": "^6.1.0",
-    "semver": "^7.3.8",
+    "semver": "^7.5.2",
```

Comment:
Although this `semver` upgrade is actually not necessary for this `commerce-sdk-react` package, we still need to do it to make sure all `semver` dependencies used by any package in the monorepo use the same version.